slate

```diff
@@ -0,0 +1 @@
+Subproject commit c90eab79f9edbfb6292c24a43c0f52a7dbbb7554
```

Comment:
Why as a submodule? Also slate docs say you should use a fork, but this commit exists in their repo, I'm confused.